desc/magnetic_fields/_current_potential.py

```diff
@@ -631,25 +769,37 @@ def _compute_magnetic_field_from_CurrentPotentialField(
         should include the potential
     basis : {"rpz", "xyz"}
         basis for input coordinates and returned magnetic field
+    compute_A_or_B: {"A", "B"}, optional
+        whether to compute the magnetic vector potential "A" or the magnetic field
+        "B". Defaults to "B"
 
 
     Returns
     -------
     field : ndarray, shape(N,3)
-        magnetic field at specified points
+        magnetic field or vector potential at specified points
 
     """
+    errorif(
+        compute_A_or_B not in ["A", "B"],
+        ValueError,
+        f'Expected "A" or "B" for compute_A_or_B, instead got {compute_A_or_B}',
+    )
     assert basis.lower() in ["rpz", "xyz"]
     coords = jnp.atleast_2d(jnp.asarray(coords))
     if basis == "rpz":
         coords = rpz2xyz(coords)
-
+    op = {"B": biot_savart_general, "A": biot_savart_general_vector_potential}[
+        compute_A_or_B
+    ]
     # compute surface current, and store grid quantities
     # needed for integration in class
+    # TODO: does this have to be xyz, or can it be computed in rpz as well?
```

Comment:
still todo? it looks like you're using rpz below?

Reply:
not todo, I can remove